Formula/a/apify-cli.rb

```diff
@@ -36,10 +36,10 @@ def install
   end
 
   test do
-    # Test that the Apify CLI is at all installed and working
-    assert_match "apify-cli/#{version}", shell_output("#{bin}/apify --version")
-    # Test that the CLI can initialize a new actor
-    system "#{bin}/apify", "init", "testing-actor"
-    assert_predicate testpath/".actor/actor.json", :exist?
+    output = pipe_output("#{bin}/apify init testing-actor", "Y")
```

Comment:
we got some issues in the automated pipeline, but this should fix the test

```suggestion
    output = pipe_output("#{bin}/apify init -y testing-actor", "Y")
```

(we no longer allow running this command in empty folder without confirmation)

Reply:
works for me 👍